casper/src/main/resources/PoS.rhox

```diff
@@ -40,22 +43,25 @@ new  PoS,
 
       getCurrentUserAddress!(*posRevAddressCh) |
       for(@posRevAddress <- posRevAddressCh) {
-        new stateCh, posVaultCh in {
+        new stateCh, posVaultCh, initialActiveCh in {
+          pickActiveValidators!($$initialBonds$$, *initialActiveCh) |
```

Comment:
I think this interpolation to make all validators active as first makes this according to plan.